prometheus/phobos-2.x/phobos-akkanet-cluster-overview-OpenTelemetry.json

```diff
@@ -1054,10 +1046,9 @@
         {
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
+            "uid": "P1809F7CD0C75ACF3"
```

Comment:
I think this is wrong - shouldn't this value be populated via a DS_PROMETHEUS variable?

Reply:
I don't know, the input portion of the json file got deleted in the export.
and there are actually 2 `pluginVersion` in the exported files

Reply:
I think the export function is broken somehow